app/presenters/tag_mapping_presenter.rb

```diff
@@ -11,11 +11,19 @@ def state_title
   end
 
   def data_attributes
-    return {} unless state == 'errored'
+    return {} unless errored?
```

Comment:
If we are showing the error in the tabe, we don't need the `data_attributes` anymore. Could you please delete those and also delete them form the view?